### ISSUE TITLE
Preference isn't showing pfm_value_unit

### ISSUE THREAD

**Issue by apizz** [erikberglund/ProfileCreator#141]
In all but one case, the `pfm_value_unit` for a preference isn't displaying.  Google Chrome - `RemoteAccessHostUdpPortRange` - excerpt below:
```
<dict>
	<key>pfm_app_min</key>
	<string>36</string>
	<key>pfm_description</key>
	<string>Restricts the UDP port range used by the remote access host in this machine.</string>
	<key>pfm_description_reference</key>
	<string>Restricts the UDP port range used by the remote access host in this machine.

If this policy is left not set, or if it is set to an empty string, the remote access host will be allowed to use any available port, unless the policy RemoteAccessHostFirewallTraversal is disabled, in which case the remote access host will use UDP ports in the 12400-12409 range.</string>
	<key>pfm_documentation_url</key>
	<string>https://dev.chromium.org/administrators/policy-list-3#RemoteAccessHostUdpPortRange</string>
	<key>pfm_name</key>
	<string>RemoteAccessHostUdpPortRange</string>
	<key>pfm_title</key>
	<string>Restrict the UDP port range used by the remote access host</string>
	<key>pfm_type</key>
	<string>string</string>
	<key>pfm_value_placeholder</key>
	<string>12400-12409</string>
	<key>pfm_value_unit</key>
	<string>range</string>
</dict>
```
![Screen Shot 2019-03-31 at 11 25 06 PM](https://user-images.githubusercontent.com/11789931/55301863-b97f9e00-540c-11e9-8731-51bc9c207fa1.png)

**Comment by erikberglund** [comment]
This is because the unit is only available to number textfields. I will add that to regular text fields as well.

**Comment by erikberglund** [comment]
It's difficult to know the width of a standard textfield, so therefore the input textfield will take the space it can, and the unit will be as small as it can.

**Comment by apizz** [comment]
that works!

**Comment by apizz** [comment]
Fixed in 0.2.5 release
